resources/var/www/html/styles/default.css

```diff
@@ -0,0 +1,37 @@
+body {
```

Comment:
```suggestion
:root {
```
use the root-element instead of body